### PR TITLE
Feat: Changed API Path to V3

### PR DESCRIPTION
**Pull request by agg3fe**
## Description
Changed API path from api/v3.0/ to api/v3/

### CODE REVIEW

**Comment by agg3fe** [comment]
@tunacicek Could you please also review these changes before we merge it.

File: docs/development/postman/tractusx-dtr-aas-3.0.0-collection.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
 								}
 							}
 						},
-						"url": "{{baseUrl}}/api/v3.0/shell-descriptors"
```
Comment (tunacicek) [inline docs/development/postman/tractusx-dtr-aas-3.0.0-collection.json:28]: Why the change in collection 3.0.0 needed?

Comment (agg3fe) [reply docs/development/postman/tractusx-dtr-aas-3.0.0-collection.json:28]: I thought if I do not change this, then it's not valid anymore. But since we have new collection, I will revert the changes in this.

**Review by tunacicek — APPROVED** [review]
LGTM
